tests/run/nogil.pyx

```diff
@@ -85,7 +85,7 @@ def test_nogil_exception_propagation():
         nogil_func()
 
 
-cdef int write_unraisable() nogil:
+cdef int write_unraisable() nogil noexcept:
```

Comment:
Based on this [discussion](https://github.com/cython/cython/pull/4670/files#r854489596), this should probably be:
```suggestion
cdef int write_unraisable() noexcept nogil:
```
(and same below several times)